apps/client/components/mainSearch/MainSearch.tsx

```diff
@@ -0,0 +1,27 @@
+import { BiSearchAlt2, BiLocationPlus } from 'react-icons/bi'
+
+import { PrimaryButton } from '../ui/PrimaryButton'
+
+export const MainSearch = () => {
```

Comment:
Tu pytanie gdzie ten mainSearch będzie używany, jeżeli tylko w jednym miejscu to imo robienie mu osobnego folderu w components to złe wyjście